PowerShellGet/PSModule.psm1

```diff
@@ -13447,6 +13496,25 @@ function Update-ModuleManifest
             $Data["ReleaseNotes"] = $($ReleaseNotes -join "`r`n")
         }
         
+        if ($PreRelease)
+        {
+            if ($ModuleVersion)
+            {
+                # if passing in a new version
+                $version = $ModuleVersion
+            }
+            else 
+            {
+                # otherwise, using existing version
+                $version = $moduleInfo.Version
+            }
+
+            if (Validate-PreReleaseString -Version $version -PreRelease $PreRelease -InfoObject $moduleInfo)
+            {
+                $Data["PreRelease"] = $PreRelease

```

Comment:
>"PreRelease" [](start = 22, length = 12)

If we are going to use this string constant multiple times, consider adding a $script: variable for this.